test/integration/storageversion/storage_version_filter_test.go

```diff
@@ -253,3 +264,40 @@ func TestStorageVersionBootstrap(t *testing.T) {
 		}
 	})
 }
+
+// assertNonStorageVersionHealthz makes sure all healthz endpoints except the
+// storage version one return ok.
+func assertNonStorageVersionHealthz(t *testing.T, cfg *rest.Config) {
+	client := clientset.NewForConfigOrDie(cfg)
+	// List all paths from root
+	discoveryPage := client.CoreV1().RESTClient().Get().AbsPath("/").Do(context.TODO())
+	body, err := discoveryPage.Raw()
+	if err != nil {
+		t.Fatalf("error getting discovery page: %v", err)
+	}
+	rootPaths := metav1.RootPaths{}
+	if err := json.Unmarshal(body, &rootPaths); err != nil {
+		t.Fatalf("error parsing root paths: %v", err)
+	}
+
+	// Check all healthz endpoints except built-in-resources-storage-version-updater
```

Comment:
can you use `/healthz?exclude=built-in-resources-storage-version-updater`?

Reply:
thanks for the pointer. I will update the PR